### PR TITLE
Accept alignmentflag and some none parents

### PR DESCRIPTION
**Pull request by clci**

### CODE REVIEW

File: PySide2-stubs/QtWidgets.pyi [hunk 1]
```diff
@@ -673,7 +673,7 @@ class QAbstractSpinBox(PySide2.QtWidgets.QWidget):
     def resizeEvent(self, event:PySide2.QtGui.QResizeEvent) -> None: ...
     def selectAll(self) -> None: ...
     def setAccelerated(self, on:bool) -> None: ...
-    def setAlignment(self, flag:PySide2.QtCore.Qt.Alignment) -> None: ...
+    def setAlignment(self, flag:PySide2.QtCore.Qt.Alignment|PySide2.QtCore.QtAlignmentFlag) -> None: ...
     def setButtonSymbols(self, bs:PySide2.QtWidgets.QAbstractSpinBox.ButtonSymbols) -> None: ...
```
Comment (bluebird75) [inline PySide2-stubs/QtWidgets.pyi:677]: PySide2.QtCore.QtAlignmentFlag sounds like a typo. It should be PySide2.QtCore.Qt.AlignementFlag (notice the extra dot at the end) no ?

I am surprised how it works at all given all this.

Comment (clci) [reply PySide2-stubs/QtWidgets.pyi:677]: I don't get why I don't get an error running Mypy. New pull request on the way.

**Comment by clci** [comment]
I'm not entirely sure, but it seems to me that the CI tests are failing due to reasons unrelated to my pull request. Am I right? What can I do?

**Comment by bluebird75** [comment]
Hi,

I took the time to fix the CI, it was unrelated to your changes.

I have just push your changes directly to the main branch, plus a few extras setAlignement() which I found (see https://github.com/python-qt-tools/PySide2-stubs/commit/94f424c6a1df08634ac74fc60ee34e6e34ac9db9 )

Thanks for the PR, you can consider it as merged.

**Pull request closed without merging**